content/news/posts/2024/PSDevCon 2024-PrestaShop-UIKit.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: PSDevCon 2024 - PUIK (PrestaShop UI Kit) - The new UI kit from PrestaShop
+subtitle: Explore a summary of the PUIK presentation delivered at the second PrestaShop Developer Conference
+date: 2024-12-12
+authors: [ PrestaShop ]
```

Comment:
The author should be @mattgoud and PrestaShop, he has to be added to authors.yml

Reply:
Hi Krystian, thanks for the note and the comment, I've added Matthias to authors on the authors.yml page.